### PR TITLE
I18n

### PR DESCRIPTION
**Pull request by TheNewCivilian**
Translating home page.
@ohrie @AlexandraKapp please check!
Making use of https://github.com/kurtsson/jekyll-multiple-languages-plugin

### CODE REVIEW

File: _i18n/en.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  team: Team
+  project: Project
+  contact: Contact
+  cargoBikeIndex: CargoBikeIndex
+  mapathon: Mapathon
+  blog: Blog
+
+team:
+  headline: Team
+  description: We are an interdisciplinary team, focusing on all about mobility. Operating as team CargoRocket since the MobiData BW Hackathon 2020.
```
Comment (ohrie) [inline _i18n/en.yml:18]: I would write:
all about sustainable mobility.

File: _i18n/de.yml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  email: team@cargorocket.de
+  twitter:
+    handle: "cargo_rocket"
+    description: Wir sind auf Twitter!
+    icon: twitter.png
+  github:
+    icon: github.png
+    url: https://github.com/CargoRocket
+    description: Wir entwickeln offen auf GitHub.
+  contactText: Du hast Anregungen, ein Unternehmen im Bereich Radlogistik oder Lust auf Zusammenarbeit? Schreib uns!
```
Comment (ohrie) [inline _i18n/de.yml:56]:
```suggestion
  contactText: Du hast Anregungen, bist ein Unternehmen im Bereich Radlogistik oder grundsätzlich Lust auf Zusammenarbeit? Schreib uns!
```

**Review by ohrie — APPROVED** [review]
Looks nice :)

**Comment by ohrie** [comment]
What about the other pages? How should we proceed?

**Comment by TheNewCivilian** [comment]
> What about the other pages? How should we proceed?

I would merge this back and continue translating our Mapathon / Blog Posts

File: _i18n/en.yml [hunk 3]
```diff
@@ -0,0 +1,60 @@
+main-headline: Cargobike Routing
```
Comment (AlexandraKapp) [inline _i18n/en.yml:1]: we should have a consistent way of writing "cargo bike". I think its two words in english (and without "-")

File: _i18n/en.yml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+second-headline: Riding cargo-bikes smooth and efficient.
```
Comment (AlexandraKapp) [inline _i18n/en.yml:2]:
```suggestion
second-headline: Riding cargo bikes smooth and efficient.
```

File: _i18n/en.yml (hunk 3 above)
Comment (AlexandraKapp) [inline _i18n/en.yml:1]:
```suggestion
main-headline: Cargo Bike Routing
```

File: _i18n/en.yml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  mapathon: Mapathon
+  blog: Blog
+
+team:
+  headline: Team
+  description: We are an interdisciplinary team, focusing on all about sustainable mobility. Operating as team CargoRocket since the MobiData BW Hackathon 2020.
+
+project:
+  headline: Project
+  description: Our mission is to make riding your cargobike as smooth and effective as possible. We consider being open minded and transparent as our core values. To give you more insight into our work we publish regularly on our Blog. Feel free to integrate our results into your project!
```
Comment (AlexandraKapp) [inline _i18n/en.yml:22]:
```suggestion
  description: Our mission is to make riding your cargo bike as smooth and effective as possible. We consider being open minded and transparent as our core values. To give you more insight into our work we publish regularly on our blog. Feel free to integrate our results into your project!
```

File: _i18n/en.yml [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+members:
+  - name: David Prenninger
+    skills: Application development and operations specialist.
+    img: david.jpg
+  - name: Alexandra Kapp
+    skills:  Geodata / GIS Expert. Giving data shape.
```
Comment (AlexandraKapp) [inline _i18n/en.yml:29]:
```suggestion
    skills:  Geodata / GIS Expert. Juggling the data.
```

File: _i18n/en.yml [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+    skills: Bike enthusiast and OpenStreetMap integrator.
+    img: henri.jpg
+
+products: 
+  - name: CargoRocket App
+    img: 
+    description: The low-emission revolution is rolling. More and more individuals as well as logistic companies are starting to pedal. Still the desperately needed infrastructure, wide and connected cycleways are missing. Our mission is to help you to find the smoothest route through all the motorized traffic madness.
+  - name: CargoBikeIndex
+    img: 
+    description: narrow cycleways, barriers or curbs might annoy cyclists but stop cargobikers. Our CargoBikeIndex categorizes all streets on their cargobike-friendliness.
```
Comment (AlexandraKapp) [inline _i18n/en.yml:41]:
```suggestion
    description: narrow cycleways, barriers or curbs might annoy cyclists but stop cargobikers. Our CargoBikeIndex categorizes all streets on their cargo-bike-friendliness.
```

File: _i18n/en.yml [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+  email: team@cargorocket.de
+  twitter:
+    handle: "cargo_rocket"
+    description: Follow us on Twitter!
+    icon: twitter.png
+  github:
+    icon: github.png
+    url: https://github.com/CargoRocket
+    description: Our Code is 100% OpenSource.
+  contactText: You have suggestions or you are a company in the field of cargobike logistics and want to collaborate? Write us!
```
Comment (AlexandraKapp) [inline _i18n/en.yml:56]:
```suggestion
  contactText: You have suggestions or you are a company in the field of cargo bike logistics and want to collaborate? Write us!
```

**Review by AlexandraKapp — APPROVED** [review]

**Pull request merged**